8.frontend-react/crm-frontend/src/screens/ProjectPage.jsx

```diff
@@ -0,0 +1,368 @@
+import React, {useEffect, useState, useRef} from 'react';
+import Header from '../components/header/Header';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import CrmApi from '../helpers/CrmApi';
+import Timer from '../components/timer/Timer';
+import CrmButton from '../components/crmButton/CrmButton';
+import '../styles/projectPage.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {faStopwatch, faUserCircle, faPaperclip, faEdit, faPlus, faCheckCircle, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+import statusMap from '../helpers/StatusMap';
+import ImgProject from '../components/imgProject/ImgProject';
+import CrmDropdown from '../components/crmDropdown/CrmDropdown';
+import Task from '../components/task/Task';
+import {DndProvider} from "react-dnd";
+import DragDrop from '../components/dragDrop/DragDrop';
+import {HTML5Backend} from 'react-dnd-html5-backend';
+import {useDispatch} from 'react-redux';
+import {start, stop} from '../reduxData/actions';
+
+function ProjectPage(props) {
```

Comment:
this is a big component
if you had the time - i would have broken it into smaller components - having each "frame" in a component.
i don't you have the time for this, but think about this when you do the dashboard screen ;)